addons/point_of_sale/views/res_config_settings_views.xml

```diff
@@ -393,7 +393,7 @@
                             <setting string="Preparation Display" help="Display orders on the preparation display" id="preparation_display">
                                 <field name="module_pos_preparation_display" widget="upgrade_boolean"/>
                             </setting>
-                            <setting string="Internal Notes" help="Add internal notes on order lines for the kitchen" id="iface_orderline_notes">
+                            <setting string="Internal Notes" help="Add Internal notes on order lines for the kitchen" id="iface_orderline_notes">
```

Comment:
It technically should be "Product" (not "Internal") as per spec. But you should just restore this diff. Don't make change in this line.